src/sys/socket/addr.rs

```diff
@@ -702,7 +702,7 @@ pub mod sys_control {
             let addr = sockaddr_ctl {
                 sc_len: mem::size_of::<sockaddr_ctl>() as c_uchar,
                 sc_family: AddressFamily::System as c_uchar,
-                ss_sysaddr: consts::AF_SYS_CONTROL as uint16_t,
```

Comment:
Not sure. `AF_SYS_CONTROL` does not seem to be used in other way than as `u16` but I never used these `AF_SYSTEM` addresses, so in doubt, probably it's better to let it as `c_int` ?